modules/claims_api/app/controllers/claims_api/v1/forms/disability_compensation_controller.rb

```diff
@@ -138,6 +139,16 @@ def validate_form_526
 
         private
 
+        #
+        # Any custom 526 submission validations above and beyond json schema validation
+        #
+        def validate_form_526_submission_values
+          return if form_attributes['claimDate'].blank?
+          return if Date.parse(form_attributes['claimDate']) <= Time.zone.today
+
+          raise ::Common::Exceptions::InvalidFieldValue.new('claimDate', form_attributes['claimDate'])
```

Comment:
Isn't necessary for this specific ticket, but I'd imagine we'd want to do something like consolidate all the logical groupings into validation methods.  

Something like ::

```
def validate_form_526_submission_values
  validate_claim_date!
  validate_some_other_random_piece!
  validate_yet_another_random_piece!
end

private

def validate_claim_date!
  return if form_attributes['claimDate'].blank?
  return if Date.parse(form_attributes['claimDate']) <= Time.zone.today
  
  raise ::Common::Exceptions::InvalidFieldValue.new('claimDate', form_attributes['claimDate'])
end
```

Reply:
yeah, probably, i can go ahead and get that setup here anyways though

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/7568/commits/37116890a4d979f8979d8dd3a0d2b56af8166bbb